pkg/spdk/engine.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/longhorn/longhorn-spdk-engine/proto/spdkrpc"
 )
 
-func SvcEngineCreate(spdkClient *spdkclient.Client, name, frontend string, replicaAddressMap map[string]string, port int32) (ret *spdkrpc.Engine, err error) {
+func SvcEngineCreate(spdkClient *spdkclient.Client, name, frontend string, bdevAddressMap map[string]string, port int32) (ret *spdkrpc.Engine, err error) {
 	if frontend != types.FrontendSPDKTCPBlockdev && frontend != types.FrontendSPDKTCPNvmf {
```

Comment:
Can we make this function private unexported? otherwise, need to do`replicaAddressMapToBdevAddressMap` in this function instead of callers to prevent redundant calls to conver replica addresses to bdev addresses.

Reply:
Do you mean `func (s *server) svcEngineCreate(...)`?

Reply:
I prefer it as well.